Sources/Plasma/PubUtilLib/plClientResMgr/plClientResMgr.cpp

```diff
@@ -134,3 +134,14 @@ plMipmap* plClientResMgr::getResource(const ST::string& resname)
 
     return resmipmap;
 }
+
+
+std::vector<ST::string> plClientResMgr::getResourceNames()
+{
+    std::vector<ST::string> names;
+
+    for (auto resource : ClientResources)
```

Comment:
I believe that this makes an unnecessary copy of the string. We may prefer `const auto&`

Reply:
I wish I were better at seeing these myself.